.github/workflows/binder-on-pr.yml

```
@@ -0,0 +1,14 @@
+name: Binder Badge
+on:
+  pull_request_target:
+    types: [opened]
+
+jobs:
+  binder:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+      - uses: jupyterlab/maintainer-tools/.github/actions/binder-link@v1
+        with:
+          github_token: ${{ secrets.github_token }}
```

I think this needs to be 
```suggestion
          github_token: ${{ secrets.GITHUB_TOKEN }}
```